TASVideos/Pages/Submissions/View.cshtml

```diff
@@ -158,7 +158,10 @@
 				<th>@Html.DisplayNameFor(m => m.Submission.Branch)</th>
 				<td>@Model.Submission.Branch</td>
 				<th>@Html.DisplayNameFor(m => m.Submission.RerecordCount)</th>
-				<td>@Model.Submission.RerecordCount</td>
+				<td>@(Model.Submission.RerecordCount > 0
```

Comment:
are we sure this is the only place we show a rerecord count on a submission?

Reply:
I think so